lte/gateway/deploy/magma_5g_agw/roles/magma-agw/files/gateway_mconfig

```diff
@@ -0,0 +1,157 @@
+{
```

Comment:
if you really want to go down this path of separate deployment role, lets eliminate duplicate config files.
The mconfig can be update at time of deployment.

Reply:
Ahh .. sorry its not required at all. I was doing some research and left this file unintentionally here in this PR. Let me remove it.

Reply:
Updated.